### ISSUE TITLE
supplier <-> client account number

### ISSUE THREAD

**Issue by szolsztor** [agricure/orderbook#11]
- [ ] Need to be able to add account number for each supplier to the client.

is this number generated by the client or the supplier? I assume the supplier? If so, would it be better to do this somewhere in the specific Supplier screen or the Client screen?

**Comment by smfuller1** [comment]
This would be generated by the supplier - needs to be in the client screen as there will be multiple numbers for each client (one from each supplier)

**Comment by szolsztor** [comment]
would you be able to order products from the supplier for that client if they do not have an account number?

**Comment by smfuller1** [comment]
yes - I don't believe suppliers use our account numbers - just their own.

**Comment by szolsztor** [comment]
Ah okay, so its an internally generated/created account number between client <-> supplier?  So you have a Client account number, and then multiple Client<->Supplier account numbers?

**Comment by smfuller1** [comment]
Yes. When we set a client up at the moment we assign a code to them (thinking about it now -its an old habit and we don't really need it). Yes then we add the supplier account number to the client (in theory, in practice they haven't all been done !)

**Comment by szolsztor** [comment]
Okay - lets talk about this more on the call

**Comment by szolsztor** [comment]
@smfuller1 you mentioned this in the last email, did you want them introduced again? we decided against it on the call we had

**Comment by smfuller1** [comment]
Is this about assigning a unique number to each client?

**Comment by szolsztor** [comment]
yeah, a unique number for each client for each supplier

**Comment by smfuller1** [comment]
yeah - that would be the clients account number per supplier
